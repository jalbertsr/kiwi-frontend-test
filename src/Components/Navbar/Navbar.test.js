import NavBar from './NavBar'
import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import { shallow, configure } from 'enzyme'

configure({ adapter: new Adapter() })

/* eslint-disable no-unused-vars */

describe('NavBar', () => {
  test('NavBar renders correctly', () => {
    const component = shallow(<NavBar />)
    expect(component).toMatchSnapshot()
  })
})
