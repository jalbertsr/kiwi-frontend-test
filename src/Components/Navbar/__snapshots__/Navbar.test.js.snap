// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar NavBar renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <MenuItem>
        <Link
          replace={false}
          to="/"
        >
          <Icon
            type="home"
          />
           Home
        </Link>
      </MenuItem>,
      "className": "",
      "mode": "horizontal",
      "prefixCls": "ant-menu",
      "theme": "light",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Link
          replace={false}
          to="/"
        >
          <Icon
            type="home"
          />
           Home
        </Link>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Icon
              type="home"
            />,
            " Home",
          ],
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "type": "home",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          " Home",
        ],
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <MenuItem>
          <Link
            replace={false}
            to="/"
          >
            <Icon
              type="home"
            />
             Home
          </Link>
        </MenuItem>,
        "className": "",
        "mode": "horizontal",
        "prefixCls": "ant-menu",
        "theme": "light",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Link
            replace={false}
            to="/"
          >
            <Icon
              type="home"
            />
             Home
          </Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Icon
                type="home"
              />,
              " Home",
            ],
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "type": "home",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            " Home",
          ],
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
